spec/fixture_game_state_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'find'
+
+require 'engine'
+require 'spec_helper'
+
+FIXTURES_DIR = File.join(File.dirname(__FILE__), 'fixtures')
+
+def full_desc(action_id, desc)
+  "at action #{action_id} #{desc}"
+end
+
+def at_action(action_id, desc)
+  it(full_desc(action_id, desc)) do
+    # share this across all individual tests within a describe block; doing this
+    # inside of an `it` prevents it from being executed if the `-e` flag is used
+    # on the commandline to run specific tests, and this test would be excluded
+    unless self.class.class_variable_defined?(:@@data)
```

Comment:
Using `@@data` directly instead of via `class_variable_*` makes rspec print out a bunch of warnings